packages/office-ui-fabric-react/src/components/Dialog/examples/Dialog.Blocking.Example.tsx

```diff
@@ -26,74 +22,61 @@ const INITIAL_OPTIONS: IComboBoxOption[] = [
-        <DefaultButton secondaryText="Opens the Sample Dialog" onClick={this._showDialog} text="Open Dialog" />
-        <Dialog
-          hidden={hideDialog}
-          onDismiss={this._closeDialog}
-          dialogContentProps={{
-            type: DialogType.normal,
-            title: 'Missing Subject',
-            subText: 'Do you want to send this message without a subject?',
-          }}
-          modalProps={{
-            isBlocking: true,
-            styles: { main: { maxWidth: 450 } },
-            dragOptions: isDraggable ? this._dragOptions : undefined,
-          }}
-        >
-          <SpinButton
-            defaultValue="0"
-            label={'Number of subjects to add:'}
-            min={0}
-            max={100}
-            step={1}
-            iconProps={{ iconName: 'IncreaseIndentLegacy' }}
-            // tslint:disable:jsx-no-lambda
-            onFocus={() => console.log('onFocus called')}
-            onBlur={() => console.log('onBlur called')}
-            incrementButtonAriaLabel={'Increase value by 1'}
-            decrementButtonAriaLabel={'Decrease value by 1'}
-          />
-          <ComboBox
-            label="Sample subject lines you could add instead"
-            placeholder="Select or type an option"
-            allowFreeform
-            autoComplete="on"
-            options={INITIAL_OPTIONS}
-          />
-          <DialogFooter>
-            <PrimaryButton onClick={this._closeDialog} text="Send" />
-            <DefaultButton onClick={this._closeDialog} text="Don't send" />
-          </DialogFooter>
-        </Dialog>
-      </div>
-    );
-  }
-
-  private _showDialog = (): void => {
-    this.setState({ hideDialog: false });
-  };
-
-  private _closeDialog = (): void => {
-    this.setState({ hideDialog: true });
-  };
-
-  private _toggleDraggable = (): void => {
-    this.setState({ isDraggable: !this.state.isDraggable });
-  };
-}
+export const DialogBlockingExample: React.FunctionComponent = () => {
+  const [hideDialog, { toggle: toggleHideDialog }] = useBoolean(true);
+  const [isDraggable, { toggle: toggleIsDraggable }] = useBoolean(false);
+  return (
+    <>
+      <Checkbox label="Is draggable" onChange={toggleIsDraggable} checked={isDraggable} />
+      <DefaultButton secondaryText="Opens the Sample Dialog" onClick={toggleHideDialog} text="Open Dialog" />
+      <Dialog
+        hidden={hideDialog}
+        onDismiss={toggleHideDialog}
+        dialogContentProps={dialogContentProps}
+        modalProps={{
+          isBlocking: true,
+          styles: modalPropsStyles,
+          dragOptions: isDraggable ? dragOptions : undefined,
+        }}
+      >
+        <SpinButton
+          defaultValue="0"
+          label={'Number of subjects to add:'}
+          min={0}
+          max={100}
+          step={1}
+          iconProps={iconProps}
+          // tslint:disable:jsx-no-lambda
```

Comment:
This should probably be:

```suggestion
          // tslint:disable-next-line:jsx-no-lambda
```

And then have this copied above `onBlur`. This will make it so that only those lines have that rule disabled instead of the rest of the file.